gwc/geowebcache-diskquota-jdbc.xml

```diff
@@ -1,5 +1,6 @@
 <gwcJdbcConfiguration>
   <dialect>PostgreSQL</dialect>
+  <JNDISource>jdbc/gwc</JNDISource>
```

Comment:
I see there's still the connectionpool info below.
This is a priori not required anymore with the JNDISource

Reply:
I took exemple from geoserver configured with gwc jndi source and it keep the information after jndisource 
but i can remove it if it is working without

Reply:
okay